File: z-clients/javascript/src/models/evictionPolicy.ts
// this file is @generated

export enum EvictionPolicy {
    NoEviction = 'no-eviction',
    }

export const EvictionPolicySerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): EvictionPolicy {
        return object;
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: EvictionPolicy): any {
        return self;
    }
}

File: z-clients/rust/src/models/eviction_policy.rs
// this file is @generated
use std::fmt;

use serde::{Deserialize, Serialize};

#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
pub enum EvictionPolicy {
    #[serde(rename = "no-eviction")]
    NoEviction,
}

impl fmt::Display for EvictionPolicy {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        let value = match self {
            Self::NoEviction => "no-eviction",
        };
        f.write_str(value)
    }
}
